server/db/models/Appointment.js

```diff
@@ -6,6 +6,9 @@ const schema = new Schema({
   eventId: {
     type: mongoose.Schema.Types.ObjectId,
   },
+  hostId: {
```

Comment:
`hostId` would be the same number id as `eventId`?

Reply:
Oh no `hostId` is the `userId`. I wanted to include it as `userId` but thought that would be confusing because for each appointment, there is a host and attendee, and it's not so intuitive to decide which one is the "user". 

But I don't know if I ended up making it more confusing lol. Let me know how you feel and we can pick the one that make more sense to everyone.

Reply:
To me, it works like this but we can discuss if there are any suggestions. 
A simple explation would be: the hostId is set to userId of the current host.

Reply:
That's actually a great explanation lol I should have included that in the comment

Reply:
I think an even more clear solution is just to call it `hostUserId` lol